CHANGELOG.md

```diff
@@ -22,7 +22,9 @@
 
 ### Signature changed
 
-- 
+- [http-client] Remove `timeout` option from the request config, use the `timeout` middleware instead.
+- [odata] Remove the `timeout` option from the request builder, user the `timeout` middleware instead
+- [openapi] Remove the `timeout` option from the request builder, user the `timeout` middleware instead
```

Comment:
[req] This should be a changeset and there needs to be an entry in the upgrade guide.